src/ClassyResource/_makeRequest.js

```diff
@@ -73,12 +75,28 @@ export default function _makeRequest(path, method, headers = {}, form, data = {}
            */
           if (this._errorLogger) {
             this._errorLogger(error, {
-              location: '_makeRequest.js',
-              action: '_makeRequest - request()',
               originalError: err,
-              response,
-              body,
-              requestParams
+              params: {
+                path,
+                method,
+                headers,
+                form,
+                data
+              },
+              /**
+               * We omit the body from the returned response
+               * since it's JSONified.
+               *
+               * In-order to make it easier for clients to filter out sensitive
+               * data we include the body as a non-JSONified object.
+               */
+              response: Object.assign(
+                {},
+                _.omit(response, 'body'),
+                {
+                  body: utils.isJson(body) ? JSON.parse(body) : body
```

Comment:
maybe make this whole weird thing a util that uses the tryParse util

Reply:
`response: jsonParseChildren(response, ['body'])`

Reply:
Let's not do that yet. I think if more of a need arises to do this then we could consider adding it.

Reply:
Okay, I did it. Not sure it makes much of a difference though haha